Bookish.Web/Controllers/HomeController.cs

```diff
@@ -26,5 +27,13 @@ public ActionResult Contact()
 
             return View();
         }
```

Comment:
Please remove any methods that you're not using 🙂 Make sure you set the default page to be one you're using. That can be whatever you prefer; the catalogue, the loans or a separate homepage with some homepage message. It's set in `Startup.cs` with `MapControllerRoute` 🙂